src/Microsoft.Framework.Runtime.Common/Impl/PlatformHelper.cs

```diff
@@ -2,18 +2,23 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.Framework.Runtime.Infrastructure;
 
 namespace Microsoft.Framework.Runtime
 {
     internal static class PlatformHelper
```

Comment:
IServiceProvider unfortunately isn't easily accessible to a lot of the locations that use PlatformHelper. For the places it is, something similar to ```if((((IRuntimeEnvironment)_services.GetService(typeof(IRuntimeEnvironment))).RuntimeType == "Mono"))``` will need to be used. And the places it isn't available we will need to use CallContextServiceLocator.